### ISSUE TITLE
Cannot install on macOS Sierra

### ISSUE THREAD

**Issue by jschwalbe** [AravisProject/aravis#53]
Followed the instructions on the README and was yet unable to install on macOS Sierra. After moving a few header files in place I finally got stumped at:
```
<snip>
  CC       arvgvinterface.lo
  CC       arvgvdevice.lo
arvgvdevice.c:39:10: fatal error: 'linux/ip.h' file not found
#include <linux/ip.h>
         ^
1 error generated.
make[2]: *** [arvgvdevice.lo] Error 1
make[1]: *** [all] Error 2
make: *** [all-recursive] Error 1
```

**Comment by EmmanuelP** [comment]
Aravis was patched a few month ago to add support for MacOS. Compilation on this platform is supposed to have  __APPLE__ defined. No idea how it is supposed to happen though, as I don't have an Apple machine at hand.

**Comment by EmmanuelP** [comment]
Any news ?

You may ant to try to ask on aravis maiiing list. Some subscribers are may be using MacOS.

https://www.freelists.org/list/aravis

**Comment by jschwalbe** [comment]
@EmmanuelP Thanks for the reply. I replied to your other question on the mailing list. I believe I was able to get it compiled, but now I'm not so sure that I actually have a GigE camera, so I'm not sure if 1) I compiled wrong 2) I'm using it wrong or 3) I have the wrong camera type!

In terms of compiling, it was ugly. Had to copy some libs from other apps. One version wouldn't allow me to go thru the configure process but then I was able to copy files from the one that did, etc. But eventually it built.

Thank you.
